john_miller/extrapoint.js

```diff
@@ -0,0 +1,3 @@
+var greet = require(__dirname + '/lib/greet');
+
+console.log(greet(process.argv[2]));
```

Comment:
Putting this functionality in its own file is not needed, but I appreciate the thought. (Feel free to use comments to point out bonus functionality if you want to make it more noticeable.)